pkg/api/customization/catalog/template.go

```diff
@@ -49,6 +112,7 @@ func TemplateFormatter(apiContext *types.APIContext, resource *types.RawResource
 
 type TemplateWrapper struct {
 	TemplateContentClient v3.TemplateContentInterface
+	TemplateVersion       v3.TemplateVersionInterface
```

Comment:
Our convention is for the client property to be plural TemplateVersions (I realize the one right above also does not follow the convention)